src/gz_TEST.cc

```diff
@@ -1118,7 +1119,7 @@ TEST(print_rotations_in_degrees, SDF)
 
   // Printing with in_degrees
   output = custom_exec_str(
-      GzCommand() + " sdf -p --precision 6 " + path + " --degrees "
+      GzCommand() + " sdf -p --precision 6 " + path + " --degrees 1"
```

Comment:
Is it not possible to make `--degrees` anot take an extra argument? I'd rather not change the usage if possible.

Edit: see my comment in `sdf_main.cc`.